doc/src/manual/faq.md

```diff
@@ -859,26 +859,36 @@ Modifying OpenBLAS settings or compiling Julia with a different BLAS library, eg
 
 ## Julia Releases
 
-### Do I want to use a release, beta, or nightly version of Julia?
-
-You may prefer the release version of Julia if you are looking for a stable code base. Releases
-generally occur every 6 months, giving you a stable platform for writing code.
-
-You may prefer the beta version of Julia if you don't mind being slightly behind the latest bugfixes
-and changes, but find the slightly faster rate of changes more appealing. Additionally, these
-binaries are tested before they are published to ensure they are fully functional.
+### Do I want to use the Stable, LTS, or nightly version of Julia?
+
+The Stable version of Julia is the latest released version of Julia,
+this is the version most people will want to run.
+It has the latest features, including improved performance.
+The Stable version of Julia will always be versioned according to SemVer as v1.x.y.
+A new minor release of Julia corresponding to a new Stable version is made approximately every 4 months after a few weeks of testing as a release candidate.
+Unlike the LTS version the a Stable version will not normally recieve bugfixes after another Stable version of Julia has been released.
+However, upgrading to the next Stable release will always be possible as each release of Julia v1.x
+will continue to run code written for earlier versions.
+
+You may prefer the LTS (Long Term Support) version of Julia if you are looking for a very stable code base.
+The LTS version of Julia will always be versioned according to SemVer as v1.0.x.
+It will continue to recieve bugfixes for the lifetime of Julia version 1, but will not recieve any new features.
+As a package developer, you may prefer to develop for the LTS version, to maximize the number of users who can use your package.
```

Comment:
I don't think this is such a good advice. Only people who are very conservative and want their existing code to run should stay on LTS and then you are likely not adding a bunch of new packages. Developing for the latest stable release should imo be the default.

Reply:
I present both alternatives. and the pros and cons of each.
It says *may*. not *should*.